google_guest_agent/osinfo/osinfo_unix.go

```diff
@@ -12,42 +12,22 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 
-package main
+//go:build unix
```

Comment:
we don't need the directive/pragma here, let's just use the file naming convention. (I'm migrating it wherever we can leaving the pragma only for cases where the convention will suit).

Reply:
unix is not an os type (like linux or windows) so file names don't work, build directives are required